streams/javahippie.json

```diff
@@ -0,0 +1,26 @@
+{
+  "twitch_handle": "javahippie",
+  "youtube_handle": "javahippie",
+  "twitter_handle": "javahippie",
+  "streamer_timezone": "CEST",
+  "schedule": [
+    {
+      "title": "BreakPoint - Functional Friday Lunch Break",
```

Comment:
I tend to also have a short explanation text with the entry, to have a few words to describe what's going to happen.

![schedule](https://user-images.githubusercontent.com/6537432/93324383-1a27b400-f816-11ea-9b83-87f84d1538dc.png)

I'd love to see that here as well.

Reply:
Added in commit, yet to be pushed